pages/accountLists/[accountListId]/contacts.page.tsx

```diff
@@ -1,28 +1,48 @@
-import React from 'react';
+import React, { useState } from 'react';
 import Head from 'next/head';
 import { useTranslation } from 'react-i18next';
 import { useRouter } from 'next/router';
 import { Box } from '@material-ui/core';
 import { ContactFilters } from '../../../src/components/Contacts/ContactFilters/ContactFilters';
 import { ContactsTable } from '../../../src/components/Contacts/ContactsTable/ContactsTable';
+import { ContactDetails } from '../../../src/components/Contacts/ContactDetails/ContactDetails';
 
 const ContactsPage: React.FC = () => {
   const { t } = useTranslation();
   const {
     query: { accountListId },
   } = useRouter();
 
+  const [contactDetailsId, setContactDetailsId] = useState<string | null>(null);
```

Comment:
I think using `undefined` as the default value is cleaner :)
```suggestion
  const [contactDetailsId, setContactDetailsId] = useState<string>();
```